### ISSUE TITLE
Variables sync

### ISSUE THREAD

**Issue by Tyberhertz** [TheLimeGlass/Skungee#42]
Hello, is it possible to syncronize a variable from server A via the bungee server to server B?
If yes, how do I put this into a script?

Example: Player A has an online time of 50 hours, which he has reached on server A, this online time should now be the same on server B.

**Comment by TheLimeGlass** [comment]
You can't do this with it's own Syntax or Configuration, but you can set up indices to grab from.
```
set {_server} to current server
set network variable {server::%{_server}%::something} to "something"

#getting
set async {_something} to network variable {server::%{_server}%::something}
```

**Comment by Tyberhertz** [comment]
I have now tried to customize a script for your script but somehow it is not yet ..

logfile:
[18:23:23] [Thread-195/INFO]: [Skungee] Recieved packet: UPDATEVARIABLES with data: proxy.playerdata.<none>.punkte with settable data: [type=string, data=[-128, 1, 49]]

Skript:
on quit:
	set network variable {proxy.playerdata.%{_uuid}%.punkte} to "%{playerdata.%{_uuid}%.punkte}%"
	set network variable {proxy.playerdata.%{_uuid}%.punkte} to "%{playerdata.%{_uuid}%.punkte}%"

```
on join:
	set async {playerdata.%{_uuid}%.punkte} to network variable {proxy.playerdata.%{_uuid}%.punkte}
	message "punkte wurden übertragen" to player

```

**Comment by TheLimeGlass** [comment]
Ok there are a few issues.

1.) A variable sync feature between servers is an enhancement, added enhancement lable.

2.) There is a bug where the current server syntax returns multiple rather than single, added bug label.

3.) In your script you provided, you're not setting the local variable `{_uuid}` before you use it, so you're setting a blank variable you will never be able to grab again, creating wasted space in the database.

**Comment by TheLimeGlass** [comment]
The multiple return error has been fixed in 1.1.0

**Comment by TheLimeGlass** [comment]
And the variable sync suggest is a duplicate of #34
